Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -358,16 +367,17 @@ def fetch_notables(service: client.Service, mapper: UserMappingObject, comment_t
         notable_incident = Notable(data=item)
         inc = notable_incident.to_incident(mapper, comment_tag_to_splunk, comment_tag_from_splunk)
         extensive_log(f'[SplunkPy] Incident data after parsing to notable: {inc}')
-        incident_id = create_incident_custom_id(inc)
+        custom_inc_id = create_incident_custom_id(inc)
+        incident_id = notable_incident.id or custom_inc_id
 
-        if incident_id not in last_run_fetched_ids:
+        if incident_id not in last_run_fetched_ids and custom_inc_id not in last_run_fetched_ids:
```

Comment:
nice,
please make suer that inside create_incident_custom_id() it is stated that this is used for cases there is no notable and therefore no id that can be used for deduplication of events.